6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,22 @@
+// --> Warmup
+
+const args = process.argv.slice(2);
+
+
+if (args.length === 0) {
+    console.log("No arguments provided. Please provide numbers as command line arguments.");
+} else {
+    const numbers = args.map(Number);
+
+    const validNumbers = numbers.filter(num => !isNaN(num));
```

Comment:
Tiny thing: I usually like wrapping the incoming function property in brackets, so it is easier to add an extra param later if needed.
```suggestion
    const validNumbers = numbers.filter((num) => !isNaN(num));
```

Reply:
But this is a nice way of checking the data! Again good job for the validation!